src/common/lib/metamask/escrow.js

```diff
@@ -74,17 +74,17 @@ export async function approveDaiStakingAmount(stakerAddress, stakingAmount) {
 
 export async function sendPaymentOrder(api, orderId, ethAccount, sellerEth) {
   const contracEscrowInterface = store.getters["metamask/contracts/getEscrowContract"]
-  const web3 = store.getters["metamask/getWeb3"]
   const currentData = await getOrdersData(api, orderId)
   const serviceId = currentData.serviceId
   const customerSubstrateAddress = currentData.customerId
   const sellerSubstrateAddress = currentData.sellerId
   const customerEthAddress = ethAccount
   const sellerEthAddress = sellerEth
   const dnaSampleTrackingId = currentData.dnaSampleTrackingId
-  const testingPrice = currentData.prices[0].value
-  const qcPrice = currentData.additionalPrices[0].value
-  const payAmount = (parseFloat(testingPrice) + parseFloat(qcPrice)).toFixed(2)
+  const testingPrice = (currentData.prices[0].value).replace(/,/g, "")
+  const qcPrice = (currentData.additionalPrices[0].value).replace(/,/g, "")
```

Comment:
Please use replaceAll instead  of using replace and regex pattern for these cases

Reply:
done, please re-review. thanks